radixdlt-core/radixdlt/src/main/java/com/radixdlt/DispatcherModule.java

```diff
@@ -162,12 +160,6 @@ public void configure() {
     bind(new TypeLiteral<EventDispatcher<MempoolRelayTrigger>>() {})
         .toProvider(Dispatchers.dispatcherProvider(MempoolRelayTrigger.class))
         .in(Scopes.SINGLETON);
-    bind(new TypeLiteral<EventDispatcher<MempoolFillerUpdate>>() {})
```

Comment:
what's the reason to move these out of DispatcherModule?

Reply:
The `MempoolFiller` and related classes are used exclusively for testing purposes. They are not configured nor used in the production code.